Chat App/View/Pub/PubChatLoginViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  PubChatLoginViewController.swift
+//  Chat App
+//
+//  Created by Amiel Jireh Cordova on 12/7/20.
+//  Copyright © 2020 Cordova, Jireh. All rights reserved.
+//
+
+import UIKit
+
+class PubChatLoginViewController: UIViewController {
+    
+    var bannerLabel: UILabel!
+    var userNameField: UITextField!
+    var passwordField: UITextField!
+    var loginButton: UIButton!
+    var registerButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationController?.isNavigationBarHidden = true
+    }
+    
+    override func loadView() {
+        view = UIView()
+        view.backgroundColor = .systemGray3
+        
+        // Top Label
+        bannerLabel = UILabel()
+        bannerLabel.translatesAutoresizingMaskIntoConstraints = false
+        bannerLabel.textAlignment = .center
+        bannerLabel.font = UIFont.systemFont(ofSize: 40)
+        bannerLabel.text = Constants.bannerLabel
+        view.addSubview(bannerLabel)
+        
+        // Username textfield
+        userNameField = UITextField(frame: CGRect(x: 20, y: 100, width: 300, height: 40))
```

Comment:
be careful of this implmentation it might break on smaller device. I would suggest using constraints instead

Reply:
Resolved in the next commit.

Reply:
Framing is handled through constraints.